ts/Extensions/Drilldown.ts

```diff
@@ -1089,22 +1140,33 @@ addEvent(Chart, 'afterShowResetZoom', function (): void {
         buttonOptions = (
             chart.options.drilldown &&
             chart.options.drilldown.drillUpButton
+        ),
+        breadcrumbsOptions = (
+            chart.breadcrumbs &&
+            chart.breadcrumbs.options
         );
 
     if (
         this.drillUpButton &&
+        this.breadcrumbs &&
         bbox &&
         buttonOptions &&
         buttonOptions.position &&
-        buttonOptions.position.x
+        buttonOptions.position.x &&
+        breadcrumbsOptions &&
+        breadcrumbsOptions.position.align === 'right' &&
+        breadcrumbsOptions.relativeTo === 'plotBox'
     ) {
-        this.drillUpButton.align({
-            x: buttonOptions.position.x - bbox.width - 10,
-            y: buttonOptions.position.y,
-            align: buttonOptions.position.align
-        },
-        false,
-        buttonOptions.relativeTo || 'plotBox'
+        const groupBox = this.breadcrumbs.group.getBBox();
+        this.breadcrumbs.group.align(
+            {
+                x: buttonOptions.position.x - bbox.width - 10,
+                y: buttonOptions.position.y,
+                width: groupBox.width,
+                align: buttonOptions.position.align
+            },
+            true,
+            breadcrumbsOptions.relativeTo
```

Comment:
Q: Shall we move the whole `"afterShowResetZoom"` logic to breadcrumbs? I don't like fact that Drilldown uses `Breadcrumbs.group.align()`. The same regarding the `"selection"` event below. Then perhaps we could refactor the `if()` clause above to an internal function (or a simple flag, that is set on init/update)?

Reply:
The code is moved. I have refactored the `if` but I have not moved it to a separate method for now.